### ISSUE TITLE
Use different vertical scale for rejected shared

### ISSUE THREAD

**Issue by malafaya** [tpruvot/yiimp#31]
I'm not sure what value is represented for the rejected shares (red line) but apparently it uses the same Y-axis scale as the hash rate. For some algos, this reduces the hashrate line graph to a pixel or two next to the horizontal axis due to the red value being "too high".
I suggest using a different scale for rejects, possibly representing that other scale on the right side of the graph.

**Comment by tpruvot** [comment]
yep, the problem actually is the diff of rejected shares is not stored in the db... so we guess diff 1 per rejected share... its on the todo list...
